src/models/transaction.rs

```diff
@@ -93,3 +94,13 @@ pub fn list_user_transactions(
         .map(|t| t.to_service())
         .collect())
 }
+
+pub fn delete_transaction_by_user_id(
+    conn: &database::DbPool,
+    user_id: &Uuid,
+) -> Result<(), TransactionModelError> {
```

Comment:
Instead of `Result<T, Error>`, you could set an alias. I personally like them, but it's just a suggestion. 

`pub type Result<T> = std::result::Result<T, TransactionModelError>;`

You should set it in the same file where you define the error, and import accordingly.

Reply:
Done.